src/index.html

```diff
@@ -1,13 +1,329 @@
+            <h3>Range & Integrated lights</h3>
+
+            <p>The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+
+            </p>
+          </article>
+
+          <article class="category">
+            <div class="category__photos">
+              <a href="#" class="category__link category__link-square">
+                <img
+                  class="category__photo"
+                  src="../src/images/hydrolic-square.png"
+                  alt="sporty-4"
+               >
+              </a>
+
+              <a href="#" class="category__link category__link-wide">
+                <img
+                  class="category__photo"
+                  src="../src/images/hydrolic-wide.png"
+                  alt="sporty-4"
+               >
+              </a>
+            </div>
+
+            <h3>Hydraulic disc brakes & Lightweight</h3>
+
+            <p>TBrakes with total stopping power the second you make contact. The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+
+            </p>
+          </article>
+
+          <div class="aaa">
+            <div class="category__block-button">
+              <a class="category__button" href="#contacts">Explore</a>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="contacts" id="contacts">
+        <h2 class="section-title section-title--align--center">Contact us</h2>
+      </section>
+
+      <div class="block__contacts">
+        <form
+          class="form form--align--left"
+          onsubmit="this.reset(); return false;"
+          method="POST"
+          action="#"
+       >
+          <div class="form__input">
+            <input
+              class="input"
+              type="text"
+              name="Name"
+              placeholder="Name"
+              required
+           >
+          </div>
+
+          <div class="form__input">
+            <input
+              class="input"
+              type="email"
+              name="email"
+              placeholder="Email"
+              required
+           >
+          </div>
+
+          <div class="form__input">
+            <div class="field__input">
+              <textarea
+                type="text"
+                class="textarea"
+                name="message"
+                placeholder="Message"
+             ></textarea>
+            </div>
+
+          </div>
+
+          <div class="form__input">
+            <button class="button" type="submit">Send</button>
+          </div>
+        </form>
+
+        <div class="contacts__content">
+          <div class="contacts__field">
+            <h4 class="contacts__title">Phone</h4>
+
+            <p class="contacts__view">
+              <a class="contact__link" href="tel:+1 234 5555-55-55">+1 234 5555-55-55</a>
```

Comment:
Bad value tel:+1 234 5555-55-55 for attribute href on element [a](https://html.spec.whatwg.org/multipage/#the-a-element): Illegal character in scheme data: space is not allowed.